test/bdd/fixtures/scripts/strapi_configure.sh

```diff
@@ -95,7 +95,7 @@ fi
 # Add user data
 result=$(curl --header "Content-Type: application/json" --header "Authorization: Bearer $token" \
    --request POST \
-   --data '{"userid":"100","name":"Foo Bar","email":"foo@bar.com","type":"user"}' \
+   --data '{"userid":"100","name":"John Smith","email":"foo@bar.com","type":"user"}' \
```

Comment:
Should we update email as well ? Something like `john.smith@example.com` ?

Reply:
done